getting-started.adoc

```diff
@@ -1,154 +1,192 @@
-
-1.  Create a blank Forms theme.
-2.  Navigate to **Drag & Drop Apps -> Forms Themes**.
-image:assets/images/select-forms-themes.png[Select Forms Themes]
-3.  Click **New Theme**.
-4.  Select the **Red Hat Theme**.
-5.  Name the theme, for example **wfm**.
-6.  Click **Create**.
-7.  Create a new project using the *WFM Demo Project* template.
-8.  Select the *Projects* header menu item.
-9.  Click **New project**.
-10. Select the *WFM Demo Project* template (under **Tech Preview**).
-11. Name the project, for example **wfm-demo**.
-12. Click **Create**.
-13. Wait until the project gets created, then click *Finish* at the
-bottom of the page. image:assets/images/new-project.png[New Project]
-14. Associate the *Forms theme* with the *WFM Demo Project*
-15. Select the *Projects* header menu item
-16. Open the *wfm-demo* project that you have created
-17. Select *Forms* from the project menu
-18. Select the *wfm* theme you have created in the *Project Theme*
-drop-down
-19. Click on *Save*
-20. Create a new MBaaS Service using the *WFM Auth Service* template.
-21. Select the *Services & APIs* header menu item.
-22. Click **Provision MBaaS Service/API**.
-23. Select the *WFM Auth Service* template (under Authentication).
-24. Name the service, for example **wfm-auth**.
-25. Click **Next**. image:assets/images/auth-service.png[Auth Service]
-26. Click **Finish**.
-27. Select the environment for initial deployment.
-28. On the *Details* page of the MBaaS service, select *Deploy* from the
-left-hand side menu.
-29. Click **Deploy Cloud App**.
-30. Create an auth policy using this new MBaaS service.
-31. Navigate to **Admin -> Auth Policies**.
-image:assets/images/auth-policy.png[Auth Policy]
-32. Click **Create**.
-33. Name the policy; eg. **wfm**.
-34. Select the *MBaaS Service* type.
-35. Select the `wfm-auth` service.
-36. Enter `/api/wfm/user/auth` as the endpoint.
-37. Select the default environment.
-38. Validate the settings using the user name `trever` and password
-`123`.
-* The response JSON should have the `status` property with value `ok`.
-39. Click **Create Auth Policy**.
-40. Associate the *WFM Auth Service* MBaaS service with the project.
-41. Select the *Services & APIs* header menu item.
-42. Select the *WFM Auth Service* created in step 3.
-43. Scroll down to the **Service Settings**, **Access Control**; select
-the project created above; eg. **wfm**.
-image:assets/images/associate-service.png[Associate Service]
-44. Click the *Save Service* button.
-45. From the left-hand side menu, select **Environment Variables**.
-46. Compare the `FH_SERVICE_AUTHORISED_PROJECTS` in the _App_ section to
-the one in the _System_ section. If they differ, click the *Push
+== Running the Raincatcher Demonstration App
+
+RHMAP provides a Raincatcher project template to help you run the demonstration.
+
+. Create a Raincatcher project as follows:
+
+..  Log in to the Studio and navigate to the *Projects* area.
```

Comment:
I'm not sure if this is a side-effect of the AsciiDoc renderer but it appears there's unnecessary paragraph tags around each list item in the output. If you look at the [previous version in markdown](https://github.com/feedhenry-raincatcher/raincatcher-documentation/blob/c992922a5e8affd6ce5c9d23fc8816cef9bb2c00/Getting-Started.md) the lists are much more compact and don't have extra paragraph tags. Is it possible to do the same here?